src/modalities/dataloader/large_file_lines_reader.py

```diff
@@ -72,27 +93,38 @@ def __len__(self) -> int:
         """
         return len(self.index)
 
-    def __getitem__(self, key: int | slice) -> str | list[str]:
```

Comment:
Why did you remove support for slices? The signature is now not consistent with BaseReader.__getitem__

Reply:
We never used slicing and this simplifies the interface. That's why I removed it. If later on we need it for some reason, we can add it back.